sites/example-project/src/components/ui/QueryViewerSupport/QueryDataTable.svelte

```diff
@@ -83,12 +83,21 @@
       dataPage = updatedSlice
     }
 
+    function handleWheel(event){
+      // abort if scroll is in x-direction
+      if (Math.abs(event.deltaX) > Math.abs(event.deltaY)) {
+        return
+      }
+
+      event.preventDefault()
+      index= Math.min(Math.max(0, index + Math.floor(event.deltaY/Math.abs(event.deltaY))), max)
```

Comment:
The scroll speed itself feel a little off when there are either lots of rows or not many:
- When there are lots of rows (eg 10,000) then you dont make enough progress for a roll of the wheel.
- When there are few rows (eg 20) then you instantly max out and end up at the bottom, it's difficult to move one row at once.

Would be nice if the scroll kind of accelerated: when you move the wheel a bit, then it advances a row or two, but if you start to drag lots (or if you are on a trackpad, swipe down a long way) then it speeds up and advances more rows.

Reply:
Does it feel off or does it feel constant? For one roll of a wheel `x` number of rows will be scrolled no matter what number of total rows are present in DataTable.

We could try to add certain logic to detect acceleration but is it really warranted, considering theres always the bottom slider to paginate more dramatically?

Reply:
@archiewood slowing down the scroll speed is doable but I would like to _point_ out that system scroll speed is controlled by the OS settings. So, even if 50% reduced scroll speed feels normal to you, it would be too slow or too fast for someone else with different tracking settings.

<img width="629" alt="Screenshot 2022-12-14 at 7 42 44 PM" src="https://user-images.githubusercontent.com/14029371/207619444-bd5ccd1d-ddd6-492f-a266-f15e5f80023e.png">


You can test this out yourself by comparing the scroll speed with a scroll wheel of a mouse and the trackpad on the same device.

<img width="658" alt="Screenshot 2022-12-14 at 7 45 48 PM" src="https://user-images.githubusercontent.com/14029371/207619497-553d3311-ba7c-418c-828f-b9d43467775e.png">



On the other hand, while tying a roll of the wheel to a percentage sounds a better solution, there is no way in my knowledge to get scroll delta in terms of scroll wheel units. So, I'm not sure how this could be implemented.

Do let me know, if I should proceed with 50% reduction of scroll speed.

Reply:
I see. Sorry for my ignorance here. The thing that's a bit puzzling UX wise is that the scroll in the table is so much faster than the scroll up and down the page.

Perhaps this illustrates it better, I'm scrolling roughly the same amount here.

![CleanShot 2022-12-14 at 09 34 17](https://user-images.githubusercontent.com/58074498/207623883-53ffbc59-ff91-4768-a578-907a57796665.gif)

If there's a way to make these feel more consistent, I'm all for it. I think 50% would probably be closer.

If you have other thoughts, let me know!